rotkehlchen/tests/unit/decoders/test_umami.py

```diff
@@ -0,0 +1,318 @@
+    user_address, timestamp, gas_amount, fee_amount, return_amount = arbitrum_one_accounts[0], TimestampMS(1728854533000), '0.0000267323', '0.00024', '62.23329'  # noqa: E501
+    assert events == [
+        EvmEvent(
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_amount)),
+            location_label=user_address,
+            notes=f'Burned {gas_amount} ETH for gas',
+            tx_hash=tx_hash,
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(fee_amount)),
+            location_label=user_address,
+            notes=f'Execution cost of {fee_amount} ETH',
+            tx_hash=tx_hash,
+            counterparty=CPT_UMAMI,
+            address=GM_USDC_WBTC_ADDRESS,
+        ), EvmEvent(
+            sequence_index=33,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.RETURN_WRAPPED,
+            asset=A_GM_USDC_WBTC,
+            balance=Balance(amount=FVal(return_amount)),
+            location_label=user_address,
+            notes=f'Return {return_amount} gmUSDC_WBTC to Umami',
+            tx_hash=tx_hash,
+            counterparty=CPT_UMAMI,
+            address=GM_USDC_WBTC_ADDRESS,
+        ),
+    ]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0x4f49D938c3Ad2437c52eb314F9bD7Bdb7FA58Da9']])
+def test_umami_withdraw_execution(
+        arbitrum_one_inquirer: 'ArbitrumOneInquirer',
+        arbitrum_one_accounts: list['ChecksumEvmAddress'],
+) -> None:
+    tx_hash = deserialize_evm_tx_hash('0xc604c2d451f0266bd2906360daca3459fd8a1a3b5b3c0d6e8c340197a665f9ee')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=arbitrum_one_inquirer, tx_hash=tx_hash)  # noqa: E501
+    user_address, timestamp, receive_amount = arbitrum_one_accounts[0], TimestampMS(1728854540000), '66.530341'  # noqa: E501
+    assert events == [
+        EvmEvent(
+            sequence_index=6,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.RECEIVE_WRAPPED,
+            asset=Asset('eip155:42161/erc20:0xaf88d065e77c8cC2239327C5EDb3A432268e5831'),
+            balance=Balance(amount=FVal(receive_amount)),
+            location_label=user_address,
+            notes=f'Receive {receive_amount} USDC from Umami',
```

Comment:
A few things. What is this? If this is a withdrawal then it shoudl say withdraw and type/subtyep shoudl be a withdrawal.

RECEIVE_WRAPPED means to receive a wrapped token. Here you receive USDC. So this is 100% wrong.